<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminTheoryWindow</class>
 <widget class="QMainWindow" name="AdminTheoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администрирование модуля &quot;Теория&quot; | RayTracing | Alpha 1.5v</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_name_theme">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Наименование темы:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_name_theme">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_number">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Номер страницы:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_num">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_theory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>621</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="html_theory">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>621</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_file">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>20</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Наименование файла:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_name_file">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_help">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>650</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Получить справку</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/help-button.png</normaloff>images/help-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_generation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Сгенерировать атрибуты для файла</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/generation-attributes-button.png</normaloff>images/generation-attributes-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>650</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Добавить файл с теоретическим материалом в базу</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/add-file-button.png</normaloff>images/add-file-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>650</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Закрыть форму администрирования модуля</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/exit-button.png</normaloff>images/exit-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_options">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>650</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Изменить настройки ввода</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/edit-lab-button.png</normaloff>images/edit-lab-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1286</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Открыть</string>
     </property>
     <addaction name="action_open_resource"/>
     <addaction name="action_open_directory"/>
    </widget>
    <addaction name="action_create"/>
    <addaction name="menu_4"/>
    <addaction name="action_save"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_manual"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Данные</string>
    </property>
    <addaction name="action_current_themes"/>
    <addaction name="action_current_files"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_create">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="action_manual">
   <property name="text">
    <string>Руководство по созданию файлов</string>
   </property>
  </action>
  <action name="action_current_themes">
   <property name="text">
    <string>Текущие темы </string>
   </property>
  </action>
  <action name="action_current_files">
   <property name="text">
    <string>Текущие файлы</string>
   </property>
  </action>
  <action name="action_open_resource">
   <property name="text">
    <string>Открыть из ресурсов</string>
   </property>
  </action>
  <action name="action_open_directory">
   <property name="text">
    <string>Открыть из директории</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
